src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php

```diff
@@ -37,7 +28,7 @@ public function setLength($length)
      */
     public function generate()
     {
-        return $this->secureRandom->nextBytes($this->length);
+        return random_bytes($this->length);
```

Comment:
However, I think if we're going to rely on `random_bytes`, we need to be explicit in the polyfill package as a dependency of ours.

We can't necessarily rely on the security component including it.